### DISCUSSION TITLE
productlistheader disappears

### DISCUSSION THREAD

**Discussion by cmdigitaluk** [openstore-ecommerce/OpenStore#273] [Q&A]
My Product page has the ProductDisplayAjaxList.cshtml template set up with the ProductListAjaxFilter.cshtml.  Whenever I click on sort or a product filter the productlistheader section disappears and just shows the ProductDisplayListBody.cshtml template, which breaks the design of the page.  I swapped back to the ClassicAjax theme to see if I have broken something when editing the Bootstrap theme but it did the same thing.

**Comment by leedavi** [comment]
The setup of the modules sound correct.

The filters are designed to use ajax, and return data of the product list (ProductDisplayListBody.cshtml ) to the page.  Could it be returning to the wrong section of the page?   The templates are a complex chain, if 1 is wring they all might fail.  One of the disadvantages of making the template system over flexible.

Here is a setup we have working,

![image](https://user-images.githubusercontent.com/5583141/173829668-e2e47c2e-045f-4f15-a577-6f78a8fade9a.png)
